R/zchunk_L201.en_nonco2.R

```diff
@@ -0,0 +1,358 @@
+    }
+    L201.delete.sectors <- paste0(L244.DeleteThermalService$region, L244.DeleteThermalService$supplysector )
+    L201.en_pol_emissions <- delete_zerodays(L201.en_pol_emissions, L201.delete.sectors)
+    L201.en_ghg_emissions <- delete_zerodays(L201.en_ghg_emissions, L201.delete.sectors)
+    L201.en_bcoc_emissions <- delete_zerodays(L201.en_bcoc_emissions, L201.delete.sectors)
+    L201.nonghg_max_reduction <- delete_zerodays(L201.nonghg_max_reduction, L201.delete.sectors)
+    L201.nonghg_steepness <- delete_zerodays(L201.nonghg_steepness, L201.delete.sectors)
+
+
+    # Produce outputs
+    L201.en_pol_emissions %>%
+      add_title("Pollutant emissions for energy technologies in all regions") %>%
+      add_units("Tg") %>%
+      add_comments("Take non-GHG emissions for the energy system, filter to out_resources") %>%
+      add_comments("in model base years, rename to regional SO2, filter to district heat,") %>%
+      add_comments("and delete sectors with zero heating/cooling degree days.") %>%
+      add_legacy_name("L201.en_pol_emissions") %>%
+      add_precursors("common/GCAM_region_names",
+                     "emissions/A_regions", "energy/A_regions",
+                     "temp-data-inject/L111.nonghg_tg_R_en_S_F_Yh",
+                     "temp-data-inject/L244.DeleteThermalService") ->
+      L201.en_pol_emissions
+
+    L201.en_ghg_emissions %>%
+      add_title("GHG emissions for energy technologies in all regions") %>%
+      add_units("Tg") %>%
+      add_comments("Take GHG emissions for the energy system, filter to out_resources") %>%
+      add_comments("in model base years, rename to regional SO2, filter to district heat,") %>%
+      add_comments("and delete sectors with zero heating/cooling degree days.") %>%
+      add_legacy_name("L201.en_ghg_emissions") %>%
+      add_precursors("common/GCAM_region_names",
+                     "energy/A_regions",
+                     "temp-data-inject/L112.ghg_tg_R_en_S_F_Yh",
+                     "temp-data-inject/L244.DeleteThermalService") ->
+      L201.en_ghg_emissions
+
+    L201.en_bcoc_emissions %>%
+      add_title("BC/OC emissions factors for energy technologies in all regions") %>%
+      add_units("Tg/EJ") %>%
+      add_comments("Take BC/OC emissions factors for energy technologies, filter to out_resources") %>%
+      add_comments("in model base years, rename to regional SO2, filter to district heat,") %>%
+      add_comments("and delete sectors with zero heating/cooling degree days.") %>%
+      add_legacy_name("L201.en_bcoc_emissions") %>%
+      add_precursors("common/GCAM_region_names",
+                     "energy/A_regions",
+                     "L114.bcoc_tgej_R_en_S_F_2000",
+                     "temp-data-inject/L244.DeleteThermalService") ->
+      L201.en_bcoc_emissions
+
+    L201.nonghg_max_reduction %>%
+      rename(max.reduction = max_reduction) %>% # no idea why old data system renamed this
```

Comment:
I think I thought it had to, but it doesn't actually matter since the csv -> xml conversion ignores column headings.